charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,1580 @@ spec:
+                            service.alpha.openshift.io/serving-cert-secret-name:
+                              type: string
+                        nodePort:
+                          type: string
+                        type:
+                          type: string
+                    authProxy:
+                      type: object
+                      properties:
+                        authenticatedEmails:
+                          type: object
+                          properties:
+                            createSecret:
+                              type: boolean
+                            data:
+                              type: string
+                            enabled:
+                              type: boolean
+                            secretName:
+                              type: string
+                        cookie:
+                          type: object
+                          properties:
+                            createSecret:
+                              type: boolean
+                            secretName:
+                              type: string
+                            seed:
+                              type: string
+                        delegateURLs:
+                          type: object
+                          properties:
+                            enabled:
+                              type: boolean
+                            policy:
+                              type: string
+                        enabled:
+                          type: boolean
+                        htpasswd:
+                          type: object
+                          properties:
+                            createSecret:
+                              type: boolean
+                            data:
+                              type: string
+                            secretName:
+                              type: string
+                        image:
+                          type: object
+                          properties:
+                            pullPolicy:
+                              type: string
+                            pullSecrets:
+                              type: array
+                            repository:
+                              type: string
+                            tag:
+                              type: string
+                        rbac:
+                          type: object
+                          required:
+                          - createAuthProxyClusterRole
+                          properties:
+                            createAuthProxyClusterRole:
+                              type: boolean
+                        resources:
+                          type: object
+                          properties:
+                            limits:
+                              type: object
+                              properties:
+                                cpu:
+                                  type: string
```

Comment:
I'm a bit worried about these cause you can put in integer values for this also normally. It seems like other operators don't specify the cpu/memory keys: https://github.com/coreos/prometheus-operator/blob/master/example/prometheus-operator-crd/prometheus.crd.yaml#L1416-L1430 Maybe we can do that?

Reply:
I originally had this as `type: integer`, but I was getting validation errors in e2e complaining that it's not `type: string`, so I changed this field, and forgot to remove this/update e2e.

Reply:
I'll update to match the prometheus-operator CRD

Reply:
Yeah, I think upstream uses https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#intorstring